core/arch/aarch64/instr_create.h

```diff
@@ -277,6 +277,15 @@
     opnd_add_flags((sht), DR_OPND_IS_SHIFT), (sha))
 #define INSTR_CREATE_b(dc, pc) \
   instr_create_0dst_1src((dc), OP_b, (pc))
+
+/**
+ * This macro creates an instr_t for a BL (branch and link) instruction.
+ * \param dc The void * dcontext used to allocate memory for the instr_t.
+ * \param pc The opnd_t target operand containing the program counter to jump to.
+ */
+#define INSTR_CREATE_bl(dc, pc) \
+  instr_create_0dst_1src((dc), OP_bl, (pc))
```

Comment:
I've updated the patch to add X30 as implicit dst operand. It requires support by the encoder/decoder though, which I created a separate PR for: #2345